lib/frontend/delete-github-subscription.js

```diff
@@ -48,24 +64,32 @@ module.exports = async (req, res) => {
       const role = await getRole({ login, installation: userInstallation });
       if (role !== 'admin') {
         res.status(401);
-        return res.json({
+        res.json({
           err: `Failed to delete subscription for ${req.body.installationId}. User does not have access to that installation.`,
         });
+        return;
       }
 
-      await Subscription.uninstall({
-        installationId: req.body.installationId,
-        host: req.body.jiraHost,
-      });
-      return res.sendStatus(202);
+      /** @type {import('../models/subscription')} */
+      const subscription = await Subscription.getSingleInstallation(req.body.jiraHost, req.body.installationId);
+      /** @type {import('../models/installation')} */
+      const installation = await Installation.getForHost(req.body.jiraHost);
+      const action = ActionFromSubscription(subscription, installation);
+      action.type = ActionType.DESTROYED;
+      action.actionSource = ActionSource.WEB_CONSOLE;
+
+      await subscription.destroy();
+      await submitProto(action);
+      res.sendStatus(202);
     } catch (err) {
       res.status(403);
-      return res.json({
+      res.json({
         err: `Failed to delete subscription to ${req.body.installationId}. ${err}`,
       });
     }
   } catch (err) {
+    console.log(err);
```

Comment:
do we want to console.log?

Reply:
err no, and the linter should've caught that, dang.

Reply:
yeah that rule appears to be off, I saw them when I did a project-wide search. I'll file an issue but I figure we can probably just leave them :p